File: JS/URI/1040.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(" ");
var nota1 = parseFloat(line[0]);
var nota2 = parseFloat(line[1]);
var nota3 = parseFloat(line[2]);
var nota4 = parseFloat(line[3]);

var media = ((nota1*2)+(nota2*3)+(nota3*4)+(nota4))/10;

if(media >= 7.0)
{
    console.log("Media: " + media.toFixed(1));
    console.log("Aluno aprovado.");
}

else if(media >=5.0 && media <= 6.9)
{
    console.log("Media: " + media.toFixed(1));
    console.log("Aluno em exame.");

    var exame = parseFloat(lines);

    var media_final = (media + exame)/2;

    if(media_final >= 5.0)
    {
        console.log("Nota do exame: "+ exame.toFixed(1));
        console.log("Aluno aprovado.");
        console.log("Media final: "+media_final.toFixed(1));
    }
    else
    {
        console.log("Nota do exame: "+ exame.toFixed(1));
        console.log("Aluno reprovado.");
        console.log("Media final: "+media_final.toFixed(1));
    }

}
else if(media < 5)
{
    console.log("Media: " + media.toFixed(1));
    console.log("Aluno reprovado.");

}

File: Python/URI/1040.py
linha = input().split(' ')

n1 = float(linha[0])
n2 = float(linha[1])
n3 = float(linha[2])
n4 = float(linha[3])

media = ((n1*2)+(n2*3)+(n3*4)+n4)/10

print("Media: {:.1f}".format(media))

if media >= 7:
    print("Aluno aprovado.")

elif media>=5 and media <=6.9:
    print("Aluno em exame.")
    exame = float(input())
    media_final = (media+exame)/2

    print("Nota do exame: {:.1f}".format(exame))

    if media_final >= 5:
        print("Aluno aprovado.")
        print("Media final: {:.1f}".format(media_final))
    
    else:
        print("Aluno reprovado.")
        print("Media final: {:.1f}".format(media_final))

else:
    print("Aluno reprovado.") 